src/components/popper/README.md

```diff
@@ -0,0 +1,40 @@
+The `PopperComponent` component is used to append an arbitrary popover panel to an on-screen element. It relies
+on the [popper.js project](https://popper.js.org/) and [`react-popper`](https://fezvrasta.github.io/react-popper/),
+which have documentation on parameters used by this component (e.g., `modifiers`).
+
+It expects two children where the first child is the reference element and the second child is the popper content.
+
+### Examples
+
+**auto placement**
+
+```js
+<div style={{position: 'relative'}}>
+    <PopperComponent isOpen>
+        <div className="flex-center-center" style={{border: "1px solid black", width: '200px', height: '100px', cursor: 'pointer'}}>Reference Element</div>
```

Comment:
I think the inline styles here make this a bit difficult to read. Can these examples share a single JS block so they can share a style variable?